src/test/java/hyperexecute/SelPlayGroundTest1.java

```diff
@@ -59,18 +59,18 @@ public void SetUpBrowser(String browserName, String version, String platform, St
         capabilities.setCapability("video", true);
         capabilities.setCapability("console", true);
 
-        try
-        {
-            driver = new RemoteWebDriver(new URL("https://" + user_name + ":" + access_key + gridURL), capabilities);
-        }
-        catch (MalformedURLException e)
-        {
-            System.out.println("Invalid grid URL");
-        }
-        catch (Exception e)
-        {
-            System.out.println(e.getMessage());
-        }
+        // try
```

Comment:
Let's not add any commented code in our public repos.

Reply:
this is actually a required code, commented it only for a demo purpose, need to record demo from the quickstart, after which this will be uncommented